exonum/src/proto/schema/exonum/proof.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+import "helpers.proto";
+
+package exonum.proof;
+
+message ListProof {
+    repeated HashedEntry proof = 1;
+    repeated ListProofEntry entries = 2;
```

Comment:
Nit: `ListProofEntry` looks like a non-descriptive name. I'd suggest something like

```proto
message ListProof {
  message HashedEntry { /* ... */ }
  message ValueEntry { /* ... */ }

  repeated HashedEntry proof = 1;
  repeated ValueEntry entries = 2;
}
```

Also, a semi-troll proposition is to use `map<uint64, bytes>` for `entries`. Are `map`s supported by the crate we use?

Reply:
Yes, it does. Will be added in the new PR.